### PR TITLE
feat: implement async sync with `unasync-cli`

### PR DESCRIPTION
**Pull request by leynier**
Run command `unasync postgrest_py tests` to generate synchronous code and tests version

Fix #27 

This PR does not use inheritance so the code is clearer

Pros: more readable code
Cons: largest library

I did another PR with using inheritance so that you accept the alternative that seems best to you: #29

### CODE REVIEW

**Comment by lqmanh** [comment]
@leynier Thank you!
Inheritance is the better way to go for the long term, but well it's just optimization. This PR is big already, we should do that in a separate PR. Furthermore, could you please add the generated code in a separate commit, for me to review more easily?

**Comment by leynier** [comment]
> @leynier Thank you! Inheritance is the better way to go for the long term, but well it's just optimization. This PR is big already, we should do that in a separate PR. Furthermore, could you please add the generated code in a separate commit, for me to review more easily?

Ok, I will do

**Comment by leynier** [comment]
@lqmanh Done ✅

**Comment by leynier** [comment]
Hi @lqmanh, can you approve running workflows?

**Comment by dreinon** [comment]
Approved

> Hi @lqmanh, can you approve running workflows?

**Comment by dreinon** [comment]
Sorry for the mess with lock file

**Comment by dreinon** [comment]
@lqmanh ready to review :slightly_smiling_face:

File: postgrest_py/utils.py [hunk 1]
```diff
@@ -1,7 +1,16 @@
+from httpx import AsyncClient  # noqa: F401
+from httpx import Client as BaseClient  # noqa: F401
+
+
+class SyncClient(BaseClient):
+    def aclose(self) -> None:
+        self.close()
```
Comment (lqmanh) [inline postgrest_py/utils.py:7]: Hm why do we need this?

Comment (leynier) [reply postgrest_py/utils.py:7]: It is a small deficiency in how the automatic generation of the synchronous code works.

If we see in this line (https://github.com/leynier/postgrest-py/blob/fc16d8bec997b7c34e0d7341ee72197d27e31f05/postgrest_py/_async/client.py#L36) of the asynchronous code (original code) when it is translated to its synchronous version (https://github.com/leynier/postgrest-py/blob/fc16d8bec997b7c34e0d7341ee72197d27e31f05/postgrest_py/_sync/client.py#L36) it continues to use the synchronous name `aclose` for the method so one way around is to implement that method in the synchronous version of the `httpx` client.

On the other hand, it is also necessary to create a `SyncClient` class, as the automatic translation of asynchronous to synchronous code changes the prefixes `Async` to `Sync`.

Comment (lqmanh) [reply postgrest_py/utils.py:7]: Ah understood

**Comment by lqmanh** [comment]
Nice work. Thank you for your contribution.

**Pull request merged**